python/paddle/distributed/auto_parallel/api.py

```diff
@@ -1425,3 +1425,65 @@ def to_static(
     dist_loader = dist_model.dist_loader
 
     return dist_model, dist_loader
+
+
+def unshard_dtensor(dist_tensor):
+    """
+    Converts a distributed tensor to its original dense tensor. ``unshard_dtensor``
+    can be treated as a reverse operation of ``shard_tensor``.
```

Comment:
if they are pair APIs, `unshard_tensor` would be more suitable.

Reply:
Only a distributed tensor (dtensor) can be unsharded, and only a dense tensor can be sharded with ``shard_tensor`` api. So, here named it ``unshard_dtensor``.